### ISSUE TITLE
Revamp of Functions page and related pages

### ISSUE THREAD

**Issue by novusnota** [tact-lang/tact#888]
They need:
* Better formatting
* More thorough description of attributes (`extends`, `mutates`, etc.)
* Mention of `receive()`, `external()`, `bounced()` with links to relevant pages
* Consistent and cohesive descriptions
* Better cross-linkage
* ...

Separate receivers and getters from all other functions — make two groups. Or three, to put getters in their own category.

**Comment by anton-trunov** [comment]
It's been almost a year since this issue has been opened

**Comment by novusnota** [comment]
> It's been almost a year since this issue has been opened

True. However, after doing the Learn Tact in Y minutes I finally know how this should look like, so it'll be one of the very next additions after finishing that.

UPD: Whoops, got a bit involved with other things, now back to this.
